add git-lfs instructions to download page

bravache opened the pull request.

bravache committed:
add git-lfs instructions to download page

bravache committed:
add punctuation

bravache commented:
@mwetter I just added a note box at the top of the download section to speficy that LFS needs to be installed before cloning the repo. I put a link to the installation steps, since it is different for Linux and Windows users.

mwetter commented:
@bravache : I should have mentioned that the changes will need to be made in the rst file at https://github.com/lbl-srg/soep/blob/master/doc-design/source/download.rst and then the html file can then be generated as described at https://github.com/lbl-srg/soep/tree/master/doc-design

bravache committed:
make html the right way

bravache commented:
Thanks @mwetter for the reference.
I updated the `download.rst` file and recompiled the document. I also updated the `readme.md` in `doc-design` with a version of `sphinxcontrib.bibtex` that is compatible with the MakeFile. With the latest version, I was getting the error referenced in this thread:
https://github.com/executablebooks/jupyter-book/issues/1137

mwetter committed:
Reformulated download instructions, corrected rst syntax errors

mwetter commented on doc-design/README.md:
```diff
@@ -14,7 +14,7 @@ and a few contributions, which can be installed by running
 pip3 install --user sphinx
 pip3 install --user sphinx_bootstrap_theme
-pip3 install --user sphinxcontrib.bibtex
+pip3 install --user sphinxcontrib.bibtex==1.0
```
@bravache : I don't think that is needed. I compiled with
```
$ python3 -m pip freeze | grep bibtex
bibtexparser==1.1.0
sphinxcontrib-bibtex==0.4.2
```

bravache replied:
Well, the issue I am trying to avoid with this line is for version > 2.0. I was running into the error described here:
https://github.com/executablebooks/jupyter-book/issues/1137, 
In particular, this comment was what solved it for me: https://github.com/executablebooks/jupyter-book/issues/1137#issuecomment-743993738

Could you check if you can still compile the document with the latest sphinxcontrib.bibtex?

mwetter reviewed (commented):
@bravache : Please see my inline comments.
I don't know what causes the error on your system. I compiled it fine on Ubuntu 18.04.

If you are fine with the changes, we can remove the bibtex==1.0 (or set to >=1.0 if needed) and then merge after recreating the files which I can do locally.

mwetter committed:
Revised requirement
